data/hai/hai reveal 4 middle.txt

```diff
@@ -0,0 +1,462 @@
+			`	"Oh! I almost forgot to tell you the good news. I found some people who know about the microtransceiver transmissions. Navy carriers have sensitive communication arrays that can pick up the transmissions. The Navy is combing their communication logs and will have some answers when we return."`
+			`	"Meet me in the spaceport when you have a bunk and 7 tons of cargo space free. Oh, and make sure you have a ramscoop. The navy will pay for this trip."`
+
+mission "Hai Rescue [A08] Scanner"
+	name "Missing Researchers"
+	description "Return to <destination> after scanning the Devil-Hide system. Payment is <payment>."
+	source "Earth"
+	waypoint "Devil-Hide"
+	passengers 1
+	cargo "advanced scanning equipment" 7
+	blocked `You need room for Giti and her <cargo>.`
+	on visit
+		dialog `Giti or her equipment are in one of your escorts. You need to wait for your escorts to arrive before landing again.`
+	to offer
+		has "Hai Rescue [A07] Logo Identification: done"
+	on offer
+		conversation
+			`Giti enters your ship after some navy workers load scanning equipment. She's eager to see first-hand the intergalactic space she's only theorized about in the past.`
+				accept
+	on enter "Devil-Hide"
+		log `Found a bizarre planet with a subterranean colony in its moon. The planet defies analysis and seems to violate the known laws of physics. There are Hai, humans, and reptiles (probably Korath) in the colony, but no evidence of Alphas. However, the scanners could only see the upper three floors due to interference from the planet.`
+		conversation
+			`Giti turns on the scanning equipment.`
+			`	"Okay, let's do the planet first..." Her voice trails off. "Wow! That's odd. We'll need a scanner about ten thousand times more powerful to scan that planet."`
+			`	 "I can see a bit of the surface though. The planet is odd in every way. It is completely inhospitable to all known forms of life. There's a huge network of caves and caverns coated with a crystal that is radiating... something the computer can't identify... and deuterium. The denizens of this system siphon the deuterium to make hyperspace fuel. Also, the planet has about a thousand times the mass it should, but somehow the gravity is around Earth normal."`
+			`	"Now the moon... hmmm... okay. It's just an ordinary chunk of rock, but the planet's radiation is disturbing the sensors. I can only see a few dozen meters below ground."`
+			`	"I found your missing researchers. They're in a facility beneath the moon's surface. There are three different species there: Hai, human, and some kind of reptile." You realize she probably means a Korath. "No Alphas though. I can only see the uppermost part of the facility. They have hydroponics, big cargo bays, docking bays for ships, and some spacious living areas. I detect 1,387 life signs spanning all age ranges, and that's just in the upper three floors of the facility."`
```

Comment:
A little strange that Giti, who is obviously very eager to learn all about that stuff, doesn't really ask any questions about the reptiles/Korath. Minor thing though.